docs/vm/access-to-the-vm.md

```diff
@@ -69,6 +69,7 @@ You can inject your access credentials via the Harvester dashboard as below:
 :::note
 
 You need to enter the VM to edit password or remove SSH-Key after deleting the credentials from the UI.
+The user to be added must already exist; otherwise, the credentials will not take effect.
```

Comment:
What do you think of combining this with bullet point 3 above?

> 3. Click to add either basic auth credentials or ssh keys. The user to be added must already exist in the guest OS. E.g., if using OpenSUSE, add `opensuse` as the user and select your ssh keys.

Reply:
Sounds great! Thanks for the comment!